surya/recognition/util.py

```diff
@@ -6,6 +7,26 @@
 from surya.common.polygon import PolygonBox
 from surya.recognition.schema import TextLine, TextWord, TextChar
 
+MATH_SYMBOLS = ["+", "-", "*", "/", "=", "^", "_", "\\"]
```

Comment:
Only missing thing I can think of is opening and closing brackets `['(', ')' '{', '}']`